### PR TITLE
Fixes Travis CI icon in README

### PR DESCRIPTION
**Pull request by athityakumar**
Fixes issue #309 

Ping @lokeshh

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # daru - Data Analysis in RUby
 
 [![Gem Version](https://badge.fury.io/rb/daru.svg)](http://badge.fury.io/rb/daru)
-[![Build Status](https://travis-ci.org/v0dro/daru.svg)](https://travis-ci.org/v0dro/daru)
+[![Build Status](https://travis-ci.org/v0dro/daru.svg)](https://travis-ci.org/SciRuby/daru.svg?branch=master)
 [![Gitter](https://badges.gitter.im/v0dro/daru.svg)](https://gitter.im/v0dro/daru?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
```
Comment (lokeshh) [inline README.md]: Remove the `.svg?branch=master`. The link shouldn't point to the picture.

**Comment by athityakumar** [comment]
@lokeshh : Made appropriate changes. It shows the build status on my fork now (https://github.com/athityakumar/daru/tree/fix-travis-icon).

**Comment by lokeshh** [comment]
Cool

**Comment by zverok** [comment]
Hm, but why Travis is failing on this, only README-changing, PR?.. Whether our current master is broken?

**Comment by athityakumar** [comment]
@zverok : Build error was due to incompatible commits 5868487aa60a4f3361a67768409240a890fe8826 &   8a4bb1cadc5d05600025f186d5c15aef74ebcdd1  - not due to README. That build error is to be fixed by PR #306.

**Comment by zverok** [comment]
OK, let's merge it, then :)

**Pull request merged**